bpfctl/src/main.rs

```diff
@@ -12,6 +12,8 @@ mod config;
 use config::config_from_file;
 use tonic::transport::{Certificate, Channel, ClientTlsConfig, Identity};
 
+const DEFAULT_BPFCTL_CONFIG_PATH: &str = "/etc/bpfctl/bpfctl.toml";
```

Comment:
I wonder should we be putting global constants like this in the `bpfd-common` package?

Reply:
I would like a common place. If I get around to selinux policy, I need to know all the files we manage. I moved to the top of the file as a place holder.